### DISCUSSION TITLE
Trying to use vscodium for development but my ubuntu server isnt connecting

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/foundry-full-course-cu#1448] [General]
I get this error as soon as i run "codium ." In ubuntu wsl cli and when i try to run it from vs codium editor( working fine with vscode)
```
Updating VS Code Server to version 27d855b2b4105a2aca2445223c8f3dc4694fbd58
Removing previous installation...
Installing VS Code Server for x64 (27d855b2b4105a2aca2445223c8f3dc4694fbd58)
Downloading: 100%
Failed
--2024-02-29 19:48:01--  https://update.code.visualstudio.com/commit:27d855b2b4105a2aca2445223c8f3dc4694fbd58/server-linux-x64/stable
Resolving update.code.visualstudio.com (update.code.visualstudio.com)... 13.107.246.68, 13.107.213.68, 2620:1ec:bdf::68, ...
Connecting to update.code.visualstudio.com (update.code.visualstudio.com)|13.107.246.68|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-02-29 19:48:06 ERROR 404: Not Found.

ERROR: Failed to download https://update.code.visualstudio.com/commit:27d855b2b4105a2aca2445223c8f3dc4694fbd58/server-linux-x64/stable to /home/cell/.vscodium-server/bin/27d855b2b4105a2aca2445223c8f3dc4694fbd58-1709216276.tar.gz

```

**Comment by EngrPips** [comment]
Hey @cell0901 , why not continue with `vscode` since it works fine for you and thats also the code editor the instructor used throughout the course. suggesting this so you don't get block by `vscodium` not working.

**Reply by cell0901** [reply]
ah actually it was just open source so u know it doesnt sends any data to microsoft. I know that data would be not so much imp but still if it could work out it would be good

**Reply by EngrPips** [reply]
Yeah, I know right. I mean you could keep it at the back of your mind to debug it and make it work but i wouldn't just like that to block from continuing with the course. Leave this open continue with the course using `vscode` as soon as i find anything that can be of help i would drop it here and tag you

**Reply by cell0901** [reply]
oh thanks man that would be helpful!
